docs/user-guide/102-policy-syntax.md

```diff
@@ -78,6 +79,13 @@ dns.server == "192.168.1.1"
 interfaces.name == capture.default-gw.interfaces.0.name
 ```
 
+### Path filter ```<pathexpression>```
+Filter out current state to include only the data matching the ```<path>```
+
+```
+dns-resolver.running // will return only the Nmstate part ofr dns resolving
```

Comment:
1. s/ofr/of.
2. `will return only the Nmstate part of dns resolving` - I don't understand what this sentence mean. What is the "Nmstate part"?